packages/fetch-api/src/option-types.ts

```diff
@@ -25,21 +25,22 @@ export interface CategoriesOptions extends PaginationOptions {}
 export interface SubcategoriesOptions extends PaginationOptions {}
 export interface RelatedOptions extends PaginationOptions {
     type?: 'gifs' | 'stickers' // no 'text' support, overrride MediaType
+    rating: Rating
 }
 
 export interface TrendingOptions extends PaginationOptions, TypeOption {
-    rating?: Rating
+    rating: Rating
 }
 
 export interface RandomOptions extends PaginationOptions {
     type?: 'gifs' | 'stickers' // no 'text' support, overrride MediaType
     tag?: string
-    rating?: Rating
+    rating: Rating
 }
 
 export interface SearchOptions extends PaginationOptions, TypeOption {
     sort?: SortTypes
-    rating?: Rating
+    rating: Rating
```

Comment:
@kylebcarlson I think we want to change this back to optional this we don't have to include rating in every request. 

<img width="1042" alt="Screenshot 2023-03-23 at 10 39 24 AM" src="https://user-images.githubusercontent.com/448767/227293858-5028cd5f-0506-470d-9b09-b20f07979597.png">


Is defining the default enough?